backend/scripts/migrate-error-segments/main.go

```diff
@@ -257,12 +257,12 @@ func buildStringQuery(fieldName modelInputs.ReservedErrorsJoinedKey, op string,
 		queryArray = append(queryArray, fmt.Sprintf("%s!=%s", fieldName, value))
 	case "matches":
 		for _, v := range val {
-			value := fmt.Sprintf("\\%s\\", v)
+			value := fmt.Sprintf("/%s/", v)
```

Comment:
Ran this in prod with this change, but wanted to officially change it since this will be the basis for the sessions migration